packages/cli/src/services/project.service.ts

```diff
@@ -15,6 +15,22 @@ import { SharedWorkflowRepository } from '@/databases/repositories/sharedWorkflo
 import { SharedCredentialsRepository } from '@/databases/repositories/sharedCredentials.repository';
 import { BadRequestError } from '@/errors/response-errors/bad-request.error';
 import { CacheService } from './cache/cache.service';
+import { License } from '@/License';
+import { UNLIMITED_LICENSE_QUOTA } from '@/constants';
+
+export class TeamProjectOverQuotaError extends Error {
```

Comment:
I'm considering whether we should extend `ApplicationError` instead, so we can set it as warning and avoid this ending up in sentry wdyt?